base/collection_xattr.go

```diff
@@ -265,7 +265,7 @@ func (c *Collection) subdocGetBodyAndXattrs(ctx context.Context, k string, xattr
 				xattrs[xattrKey] = xattr
 			}
 
-			if len(xattrErrors) == len(xattrs) {
+			if len(xattrErrors) == len(xattrKeys) {
```

Comment:
unrelated - @torcolvin I think I mentioned this to you the other day, would like your opinions on whether this needs fixing in another PR